girder/__init__.py

```diff
@@ -226,8 +226,8 @@ def _setupLogger():
 logger = _setupLogger()
 
 
-def logStdoutStderr():
-    if _originalStdOut == sys.stdout:
+def logStdoutStderr(force=False):
```

Comment:
Why is this flag required?

Reply:
pytest hijacks sys.stdout and sys.stderr, so `_originalStdOut` in this module is no longer equal to `sys.stdout` at test time. This flag seemed like the simplest way to workaround that.